src/node_api.cc

```diff
@@ -3088,6 +3088,20 @@ napi_status napi_get_version(napi_env env, uint32_t* result) {
   return napi_clear_last_error(env);
 }
 
+napi_status napi_get_node_version(napi_env env,
+                                  napi_node_version** result) {
+  CHECK_ENV(env);
+  CHECK_ARG(env, result);
+  static napi_node_version version = {
+    NODE_MAJOR_VERSION,
+    NODE_MINOR_VERSION,
+    NODE_PATCH_VERSION,
+    NODE_RELEASE
+  };
+  *result = &version;
```

Comment:
Since we're returning a pointer to this static struct, should the `result` be a `const napi_node_version**` to prevent modification by the callers?

I think the other option would be to use a single indirection to a non-const `napi_node_version` and then just fill in the values.  I'm fine either way.